src/views/Settings.vue

```diff
@@ -12,7 +12,7 @@
         <ion-card>
           <ion-item lines="full">
             <ion-avatar slot="start" v-if="userProfile?.partyImageUrl">
-              <Image :src="userProfile.partyImageUrl"/>
+              <ShopifyImg :src="userProfile.partyImageUrl"/>
```

Comment:
We should not make changes in the settings page as the image is not fetched from Shopify.

Reply:
reverted the changes